rollup.config.js

```diff
@@ -81,6 +81,8 @@ export default {
     css({ output: "bundle.css" }),
     // only use google analytics on production builds
     replace({
+      __GLEAN_APPLICATION_ID__:
+        process.env.GLEAN_APPLICATION_ID || "glean-dictionary-dev",
```

Comment:
We will need to set the correct app id on production @wlach. I believe this requires some tinkering on Netlify directly, correct?

Reply:
Correct, it's an easy change to make via the configuration panel. /cc @Iinh